netbeans.apache.org/src/content/download/nb14/index.asciidoc

```diff
@@ -33,37 +33,48 @@ for important requirements for download pages for Apache projects.
+
+//// 
+url-status depends of release status archived or not
+https://archive.apache.org/dist/netbeans/  (//archived)
+https://www.apache.org/dyn/closer.cgi/netbeans/ (//current)
+////
+
+Apache NetBeans {netbeans-version} was released on June 9, 2022. link:https://github.com/apache/netbeans/releases/tag/{netbeans-version}[Go here on GitHub] for a list of fixed issues for Apache NetBeans {netbeans-version}.
 
 ////
 NOTE: It's mandatory to link to the source. It's optional to link to the binaries.
 NOTE: It's mandatory to link against https://www.apache.org for the sums & keys. https is recommended.
 NOTE: It's NOT recommended to link to github.
 ////
-Apache NetBeans 14 is available for download from your closest Apache mirror.
+Apache NetBeans {netbeans-version} is available for download from your closest Apache mirror.
 
 - Binaries: 
-link:https://www.apache.org/dyn/closer.cgi/netbeans/netbeans/14/netbeans-14-bin.zip[netbeans-14-bin.zip] (link:https://downloads.apache.org/netbeans/netbeans/14/netbeans-14-bin.zip.sha512[SHA-512],
-link:https://downloads.apache.org/netbeans/netbeans/14/netbeans-14-bin.zip.asc[PGP ASC])
+link:{url-status}netbeans/{netbeans-version}/netbeans-{netbeans-version}-bin.zip[netbeans-{netbeans-version}-bin.zip] (link:{fixed-download-url}netbeans/{netbeans-version}/netbeans-{netbeans-version}-bin.zip.sha512[SHA-512],
+link:{fixed-download-url}netbeans/{netbeans-version}/netbeans-{netbeans-version}-bin.zip.asc[PGP ASC])
 
 - Installers:
  
-* link:https://www.apache.org/dyn/closer.cgi/netbeans/netbeans-installers/14/Apache-NetBeans-14-bin-windows-x64.exe[Apache-NetBeans-14-bin-windows-x64.exe] (link:https://downloads.apache.org/netbeans/netbeans-installers/14/Apache-NetBeans-14-bin-windows-x64.exe.sha512[SHA-512],
-link:https://downloads.apache.org/netbeans/netbeans-installers/14/Apache-NetBeans-14-bin-windows-x64.exe.asc[PGP ASC])
-* link:https://www.apache.org/dyn/closer.cgi/netbeans/netbeans-installers/14/Apache-NetBeans-14-bin-linux-x64.sh[Apache-NetBeans-14-bin-linux-x64.sh] (link:https://downloads.apache.org/netbeans/netbeans-installers/14/Apache-NetBeans-14-bin-linux-x64.sh.sha512[SHA-512],
-link:https://downloads.apache.org/netbeans/netbeans-installers/14/Apache-NetBeans-14-bin-linux-x64.sh.asc[PGP ASC])
-* link:https://archive.apache.org/dist/netbeans/netbeans-installers/14/Apache-NetBeans-14-bin-macosx.dmg[Apache-NetBeans-14-bin-macosx.dmg] (link:https://archive.apache.org/dist/netbeans/netbeans-installers/14/Apache-NetBeans-14-bin-macosx.dmg.sha512[SHA-512],
-link:https://archive.apache.org/dist/netbeans/netbeans-installers/14/Apache-NetBeans-14-bin-macosx.dmg.asc[PGP ASC])
+* link:{url-status}netbeans-installers/{netbeans-version}/Apache-NetBeans-{netbeans-version}-bin-windows-x64.exe[Apache-NetBeans-{netbeans-version}-bin-windows-x64.exe] (link:{fixed-download-url}netbeans-installers/{netbeans-version}/Apache-NetBeans-{netbeans-version}-bin-windows-x64.exe.sha512[SHA-512],
```

Comment:
Reverse of previous comment.  We have previously pointed SHA and PGP to the archive as well (until NB13). It does look like the download.a.o links for NB13 redirect automatically.  Not sure that has always been the case.

Reply:
well github prune my comment, for KEYS hardcodeurl for pgp-checksum url-download-keychecksum https://archive.apache.org/dist/netbeans/  (//archived)
https://downloads.apache.org/netbeans/ (//current)
and for url-download
https://archive.apache.org/dist/netbeans/  (//archived)
https://www.apache.org/dyn/closer.cgi/netbeans/ (//current)

Reply:
rebased for merging